Src/CSharpier.Tests/FormattingTests/TestFiles/cs/AssignmentExpressions.test

```diff
@@ -69,18 +69,18 @@ class ClassName
                 );
     }
 
-    public SomeExpressionBodyMethod() =>
-        nonChainFormatting = anotherVariable1 = (
-            someCondition________________________________
-            || someOtherCondition___________________________
-        );
-
-    public SomeExpressionBodyMethod() =>
-        chainFormatting =
-            anotherVariable1 =
-            anotherVariable2 =
-                (
-                    someCondition________________________________
-                    || someOtherCondition___________________________
-                );
+    public SomeExpressionBodyMethod()
+        => nonChainFormatting = anotherVariable1 = (
+                someCondition________________________________
+                || someOtherCondition___________________________
+            );
```

Comment:
There is an extra space before the closing parenthesis here (check against where `nonChainFormatting` starts above) which freaks me out a bit.